qa/evil-tests/src/test/java/org/elasticsearch/common/logging/EvilLoggerTests.java

```diff
@@ -111,7 +111,7 @@ public void testDeprecationLogger() throws IOException, UserException {
 
         final int deprecatedIterations = randomIntBetween(0, 256);
         for (int i = 0; i < deprecatedIterations; i++) {
-            deprecationLogger.deprecated("This is a deprecation message");
+            deprecationLogger.deprecatedAndMaybeLog("key" + i, "This is a deprecation message");
```

Comment:
Doesn't this test now overlap with the maybelog test further down in the file?

Reply:
Good call. I compared them, and the removed this test.